content/docs/2.15/scalers/azure-storage-queue.md

```diff
@@ -26,6 +27,7 @@ triggers:
 
 - `queueName` - Name of the queue.
 - `queueLength` - Target value for queue length passed to the scaler. Example: if one pod can handle 10 messages, set the queue length target to 10. If the actual number of messages in the queue is 30, the scaler scales to 3 pods. (Default: `5`, Optional)
+- `queueLengthStrategy` - `default` considers both visible and invisible messages, while `visibleonly` uses Peek to count only visible messages. In `visibleonly`, if the count of messages is 32 or higher, it falls back to the default strategy, counting both visible and invisible messages.
```

Comment:
Default is not really clarifying what it includes. Instead I would propose to call it `all`

Reply:
Also, please align with the "(Default: `5`, Optional)" convention as per https://github.com/kedacore/keda-docs#writing-documentation-for-a-scaler

Reply:
Thanks for the feedback. Hope this version is better?